harvester_e2e_tests/integrations/test_upgrade.py

```diff
@@ -750,6 +796,10 @@ def test_verify_logging_pods(self, api_client):
                 "\n".join(f"Pod({n})'s phase({p}) is not expected." for n, p in fails)
             )
 
+        # teardown
+        if api_client.cluster_version.release >= (1, 2, 0):
```

Comment:
to notice that if we didn't enable addon before upgrade (consider that the addon already be enabled before testing), we should not disable it.

Reply:
Enhanced in 6a19ea8 by logging_addon fixture.